.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```
@@ -8,7 +8,8 @@ body:
   - type: markdown
     attributes:
       value: |
-        Thanks for taking the time to fill out this bug report!
+        Thanks for taking the time to fill out this bug report! 
+        If you are experiencing an issue after upgrading to a new version. Please make sure you restart Iron Fish node after upgrade. You can check this by running `ironfish status` and `ironfish debug`, compare the Iron Fish version in both output, they should match.
```

A couple grammar improvement suggestions:
```suggestion
        If you are experiencing an issue after upgrading to a new version, please make sure you restart your Iron Fish node. You can check this by running `ironfish status` and `ironfish debug` and comparing the Iron Fish version. In both commands, the version should match.
```